From f0345dcaf11477ec8fb952b9f99dce1bcd0451d9 Mon Sep 17 00:00:00 2001
From: Christian Bruel <christian.bruel@foss.st.com>
Date: Mon, 30 Jan 2023 20:08:11 +0100
Subject: [PATCH 0381/1141] dt-bindings: phy: stm32-combophy: Add optional
 impedance control

Add st,output-micro-ohms and st,output-vswing-microvolt bindings entries to tune
the impedance and voltage swing using discrete simulation results

Signed-off-by: Christian Bruel <christian.bruel@foss.st.com>
Change-Id: I40845dca39962146ee767a72ed7863255115fc2f
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/313752
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
---
 .../bindings/phy/st,stm32-combophy.yaml        | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/Documentation/devicetree/bindings/phy/st,stm32-combophy.yaml b/Documentation/devicetree/bindings/phy/st,stm32-combophy.yaml
index 511d7c6b71ba..67dd9a82291f 100644
--- a/Documentation/devicetree/bindings/phy/st,stm32-combophy.yaml
+++ b/Documentation/devicetree/bindings/phy/st,stm32-combophy.yaml
@@ -66,6 +66,24 @@ properties:
       A boolean property whose presence indicates that the SSC for common clock
       needs to be set.
 
+  st,output-micro-ohms:
+    minimum: 3999000
+    maximum: 6090000
+    default: 4968000
+    description:
+      A value property to tune the PCIe 100Mhz Single Ended Output Impedance, simulations
+      results at 25C for a VDDP=0.8V.
+      The hardware accepts discrete values in this range.
+
+  st,output-vswing-microvolt:
+    minimum: 442000
+    maximum: 803000
+    default: 803000
+    description:
+      A value property in microvolt to tune the PCIe 100Mhz differential Single Ended Output
+      Voltage Swing to change the Vlo, Vhi for a VDDP = 0.8V.
+      The hardware accepts discrete values in this range.
+
 required:
   - compatible
   - reg
-- 
2.39.2

